stock-portfolio-tracker/UI/View/PimaryButton.swift

```diff
@@ -8,15 +8,27 @@
 import SwiftUI
 
 struct PrimaryButton: ButtonStyle {
+    
+    let disabled: Bool
+    
+    init() {
```

Comment:
I do not want this small change to propagate to every single button that uses `PrimaryButton`, so I've created an empty init. This maintains the previous interface and makes it so that we don't have to change code in other places.